src/test/java/nextstep/subway/acceptance/StationAcceptanceTest.java

```diff
@@ -69,8 +64,22 @@ void createStation() {
     @DisplayName("지하철역을 조회한다.")
     @Test
     void getStations() {
-    }
+        // given
+        Map<String, String> params1 = new HashMap<>();
+        params1.put("name", "강남역");
+
+        Map<String, String> params2 = new HashMap<>();
+        params2.put("name", "양재역");
+
+        createStation(params1);
+        createStation(params2);
 
+        // when
+        List<Station> stations = getStationList();
+
+        // then
+        assertThat(stations.size()).isEqualTo(2);
```

Comment:
생성한 지하철역이 응답 되는지도 검증해보면 어떨까요?

size 검증은 이렇게도 가능합니다 😄 
```suggestion
        assertThat(stations).hasSize(2);
```